ethexe/common/Cargo.toml

```
@@ -35,3 +39,4 @@ std = [
     "k256/std",
     "alloy-primitives/std",
 ]
+mock = ["std", "itertools", "itertools/use_std", "tap"]
```

it's enabled as a dep automatically 
```suggestion
mock = ["std", "itertools/use_std", "tap"]
```